src/legacy/core_plugins/metrics/server/lib/vis_data/series/get_request_params.js

```diff
@@ -16,20 +16,24 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 import buildRequestBody from './build_request_body';
 
-export default (req, panel, series) => {
-  const indexPattern = series.override_index_pattern && series.series_index_pattern || panel.index_pattern;
+export default (req, panel, series, isBatchRequest = true) => {
   const bodies = [];
 
+  if (isBatchRequest) {
+    const indexPattern = series.override_index_pattern && series.series_index_pattern || panel.index_pattern;
+
+    bodies.push({
+      index: indexPattern,
+      ignoreUnavailable: true,
+    });
+  }
+
   bodies.push({
-    index: indexPattern,
-    ignoreUnavailable: true,
+    ...buildRequestBody(req, panel, series),
+    timeout: '90s'
```

Comment:
@timroes  @markov00  
Am I right that we should use shardTimeout here instead of hardcoded 90s

See the following link for more details:
https://github.com/elastic/kibana/issues/17576

Reply:
Yes you are right. To have a clean history it's better to add this fix on a different PR